src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthenticationExtensions.cs

```diff
@@ -27,7 +30,17 @@ public static class AddAuthenticationExtensions
             this IServiceCollection services,
             OAuth2IntrospectionOptions oAuth2IntrospectionOptions)
         {
+            var nisCodeServiceUrl = services
```

Comment:
Can't we pass IConfiguration as parameter to the method?

Reply:
we could, but do we want to pollute the call interface?